src/cljc/matcher_combinators/matchers.cljc

```diff
@@ -54,3 +54,7 @@
   "Matcher that will match when given value matches the `expected` regular expression."
   [expected]
   (core/->Regex expected))
+
+(def absent
```

Comment:
should it be `absent?` then?

Reply:
the other matchers, like `embeds` and `equals` don't have the `?` so I figured I'd keep the same pattern